TramsDataApi/DatabaseModels/SipEducationalperformancedata.cs

```diff
@@ -44,5 +44,7 @@ public partial class SipEducationalperformancedata
         public decimal? SipProgress8ebaccdisadvantaged { get; set; }
         
         public int? SipPerformancetype { get; set; }
+        public decimal? SipAppliedgeneralaveragepspe { get; set; }
+        public decimal? SipAcademiclevelaveragepspe { get; set; }
```

Comment:
I think these might have gotten a bit messed by EF when generating the model. Think they should probably be camel-cased

Reply:
Yeah looks like. I'll change it 👍